spec/controllers/mixins/ems_common/angular_spec.rb

```diff
@@ -85,25 +85,25 @@
 
       it "returns connect options for openstack cloud default tab" do
         @params[:cred_type] = "default"
-        @ems_cloud_controller.instance_variable_set(:@_params, @params)
+        @ems_cloud_controller.params = @params
 
         expected_connect_options = ["v2:{XpADRTTI7f11hNT7AuDaKg==}",
                                     {:default_security_protocol => "ssl",
                                      :default_hostname          => "host_default",
                                      :default_api_port          => "13000",
-                                     :default_userid            => "abc"}]
+                                     :default_userid            => "abc"}.with_indifferent_access]
```

Comment:
we deep symbolized keys before but don't anymore... we should symbol/string keys as they shouldn't matter with the getter methods.